src/main/java/sg/ncl/testbed_interface/Team2.java

```diff
@@ -1,14 +1,19 @@
 package sg.ncl.testbed_interface;
 
+import lombok.Getter;
+import sg.ncl.domain.MemberStatus;
+
 import javax.validation.constraints.Size;
 import java.io.Serializable;
+import java.lang.reflect.Member;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.lang.reflect.Member'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)